src/RestControllers/PatientRestController.php

```diff
@@ -58,6 +58,18 @@ public function post($data)
         return RestControllerHelper::handleProcessingResult($processingResult, 201);
     }
 
+
+    /**
+     * Process a HTTP POST request used to create a patient portal credentials.
+     * @param $data - array of patient fields, $puuidString - uuid of patient
+     * @return a 200/Ok status code and the patient identifier if successful.
+     */
+    public function setPatientAccess($puuidString, $data)
```

Comment:
Try to use type hinting for the parameters the receive the method.
Ex.
public function setPatientAccess(string $puuidString, array $data): RestControllerHelper